src/dmd/func.d

```diff
@@ -2275,21 +2275,20 @@ extern (C++) class FuncDeclaration : Declaration
      * Return the function's parameter list, and whether
      * it is variadic or not.
      */
-    final Parameters* getParameters(int* pvarargs)
+    final ParameterList getParameterList(int* pvarargs)
```

Comment:
Needs to update the corresponding header file.

Reply:
Is it even used by the C++ code? @ibuclaw @kinke ?

Reply:
The function exists in a header file, not sure if it's used. It's now out of sync.

Reply:
Looks like LDC doesn't use it, but if it's in the header, it should be adapted too.

Reply:
updated .h files